README.md

```diff
@@ -359,6 +359,18 @@ conn = trino.dbapi.connect(
 )
 ```
 
+You could also pass `system` role without explicitly specifing "system" catalog:
+
+```python
+import trino
+conn = trino.dbapi.connect(
+    host='localhost',
+    port=443,
+    user='the-user',
+    roles="roleC" # equivalent to {"system": "roleC"} 
```

Comment:
I would advise against a mixed case example unless we are full sure it works in all cases.

Reply:
good point.